test/sender.js

```diff
@@ -235,6 +311,20 @@ class TestServer {
     setImmediate(() => this.on(event, handler))
   }
 
+  _onRelpData(socket, data) {
+    socket._relpInput += data.toString();
+    let m;
+    //console.log(`input:\n[${socket._relpInput}]`);
+    while((m = socket._relpInput.match(RELP_COMMAND_REGEX))) {
```

Comment:
```suggestion
    while((let m = socket._relpInput.match(RELP_COMMAND_REGEX)) !== null) {
```
and then remove the `let m;` declaration from above

Reply:
As I said in a previous reply, declaration is not allowed within the `while` condition.

d80832e